### ISSUE TITLE
Opening MFXPopup based controls upwards

### ISSUE THREAD

**Issue by gilad4** [palexdev/MaterialFX#188]
I have noticed that controls which use MFXPopup (such as: MFXDatePicker, MFXComboBox, MFXFilterComboBox, etc.), 
do not automatically open upwards when hitting the bottom screen, nor can it be done manually.

As can be seen in the demo, Legacy ComboBox does it automatically.

![image](https://user-images.githubusercontent.com/19804616/165465382-7d0184a7-c22f-4721-b7f1-92f5ba2057a0.png)

Doing it manually in a plain MFXPopup is possible but not very straight forward, still there's probably no way to reach it in the mentioned pickers without involving dirty code.

**Comment by palexdev** [comment]
Yes I'm well aware of the issue. Thanks for reminding me, I'll try to fix it for the next release

**Comment by gilad4** [comment]
Hi @palexdev,

Thank you for working on the issue. I appreciate your effort in trying to fix it.

However, the expected behavior is still not quite there. I would expect the popup to open upwards without covering the owner dropdown control, similar to how it works on the legacy controls. With the current change, the popup open upwards but still covers the owner control.

Let me know what you think.

**Comment by palexdev** [comment]
I don't know why though, I just flipped the JavaFX default switch, maybe it is the positioning algorithm, I have no clue honestly will see what I can do, but I don't assure you I can improve it further
